src/scss/components/checkout/checkoutRemote/_checkoutRemote.scss

```diff
@@ -7,6 +7,10 @@
     flex-wrap: wrap;
     margin-left: -(spacing("third"));
     margin-right: -(spacing("third"));
+
+    > div {
+        padding-right: spacing("single");
```

Comment:
I think `margin-right` would be more appropriate for this.

Reply:
Updated to use margin instead. Confirmed button spacing appears the same in VM.